src/applications/mhv-landing-page/components/HeaderLayout.jsx

```diff
@@ -58,9 +59,12 @@ const HeaderLayout = ({
             manage your care.
             <br />
             <a
-              onClick={() =>
-                datadogRum.addAction('Click on My VA Health portal link')
-              }
+              onClick={() => {
+                datadogRum.addAction('Click on My VA Health portal link');
+                recordEvent({
+                  event: 'va-health-portal-link',
+                });
```

Comment:
I'm not sure how GA is currently tracking new event. If it’s not being picked up, maybe we could try using the 'nav-link-click' event instead?
Example:
```js
recordEvent({
  event: 'nav-link-click',
  action: 'click',
  'link-label': 'Go to the My VA Health portal',
  'link-destination': <<destination link>>,
  'link-origin': window.location.href,
});
```